fvm/evm/precompiles/arch.go

```diff
@@ -25,8 +27,11 @@ var (
 	// but we might increase this in the future
 	ProofVerifierGasMultiplerPerSignature = uint64(3_000)
 
-	// RandomSourceGas covers the cost of calculating a revertible random bytes
-	RandomSourceGas = uint64(1_000) // todo define
+	// RandomSourceGas covers the cost of obtaining random sournce bytes
+	RandomSourceGas = uint64(1_000)
+
+	// RevertibleRandomGas covers the cost of calculating a revertible random bytes
+	RevertibleRandomGas = uint64(1_000)
```

Comment:
🤔 I wonder if we need these in the future. The cadence gas cost for calling randomness should probably already be enough.

Reply:
It adds some overhead on the cost, I think that is reasonable since the call has to be "routed" to the FVM which takes up some computation effort. But probably is so small I can agree to remove. Maybe we can revisit at some later point and do some calibration.